node/utils/validation.ts

```diff
@@ -23,7 +24,7 @@ export const isShippingValid = async (
 
   const address = shipping.selectedAddress
 
-  if (!address.country) {
+  if (!address?.country) {
```

Comment:
What do you think of testing if `address` (or `shipping.selectedAddress`) is `null` before this? It might seem redundant, but I believe it is safer to be explicit.

idk, I just feel like someone might break stuff by adding something in line 26.

Reply:
sure